data/templates/Detector_Automation.csx

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+		let startTime = datetime({cxt.StartTime});
+		let endTime = datetime({cxt.EndTime});
+		cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
+		| where Timestamp == startTime and Timestamp == endTime
```

Comment:
Please change Timestamp to TIMESTAMP, most Kusto tables have this column name in uppercase. We should also update our OneNote doc and make it uppercase there.

Reply:
Updated the OneNote ;)